third_party/terraform/tests/resource_logging_bucket_config_test.go

```diff
@@ -77,6 +77,41 @@ func TestAccLoggingBucketConfigProject_basic(t *testing.T) {
 	})
 }
 
+func TestAccLoggingBucketConfigProject_customBucketId(t *testing.T) {
```

Comment:
Any way we can test creation for all bucket types? There might be a nice way to loop through this. Otherwise multiple tests will do

Reply:
added the tests. please have a look